torch/utils/tensorboard/__init__.py

```diff
@@ -1,6 +1,9 @@
-try:
-    from tensorboard.summary.writer.record_writer import RecordWriter  # noqa F401
-except ImportError:
-    raise ImportError('TensorBoard logging requires TensorBoard with Python summary writer installed. '
-                      'This should be available in 1.14 or above.')
+import tensorboard
+from distutils.version import LooseVersion
+if LooseVersion(tensorboard.__version__) < LooseVersion('1.15'):
+    raise ImportError('TensorBoard logging requires TensorBoard version 1.15 or above')
```

Comment:
If TensorBoard is not installed, will this error message show?

Reply:
`import tensorboard` would fail with a normal `ImportError`.